packages/graphql-hooks-mock-client/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "graphql-hooks-mock-client",
+  "version": "0.1.0",
+  "description": "A mock client for graphql-hooks",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "scripts": {
+    "clean": "rm -rf ./dist ./es ./lib",
+    "test": "echo \"Error: no test specified\" && exit 0",
+    "prepublishOnly": "cp ../../LICENSE ."
+  },
+  "keywords": [
+    "graphql",
+    "hooks",
+    "react",
+    "graphql-hooks",
+    "cache",
+    "mocks",
+    "mock client"
+  ],
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "graphql-hooks": "^5.5.0"
```

Comment:
I though that a `peerDependency` would be the best choice but `lerna bootstrap` doesn't automatically link `graphql-hooks` in the `node_modules`.

I've made a workaround by manually creating the link inside the `packages` dir, but maybe it would be easier to move the `graphql-hooks` inside the dependencies.
That would also enable the automatic version bump when we do a change on `graphql-hooks`.

Does someone know what is the best pratice here?